docs/content/management-bundles.md

```diff
@@ -941,6 +941,9 @@ bundles:
     resource: 'bundle.tar.gz?alt=media'
 ```
 
+If the resource (the object in the gcs bucket) contains slashes (/) or other special characters, these need to be url-encoded here, e.g.
+`bundles/bundle.tar.gz?alt=media` should be entered as `bundles%2fbundle.tar.gz?alt=media`.
```

Comment:
A link to https://cloud.google.com/storage/docs/request-endpoints#encoding would be good here.

Reply:
thanks, I added the link